Modules/Quest/DailyQuests.lua

```diff
@@ -205,4 +249,85 @@ pvpDailyIds = {
     [11340] = true,
     [11341] = true,
     [11342] = true,
-}
\ No newline at end of file
+}
+
+everydayDailyIds = {
```

Comment:
I don't think it is a good idea to have all these IDs in here. They should come from the database and most likely be checked by the `questFlags` field as suggested in #3161

Reply:
I hope they will come in future from DB, so that `everydayDailyIds` (or renamed `generalDailyIds`) is auto generated at database recompile. As well as other dailyIds.